src/components/generics/vacation-test-forms/vacationRequestForm.tsx

```diff
@@ -0,0 +1,217 @@
+import { CalendarPickerView } from "@mui/x-date-pickers";
+import { Box, Button, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, TextField, Typography } from "@mui/material";
+import { VacationType } from "generated/client";
+import strings from "localization/strings";
+import { ChangeEvent, useEffect, useState } from "react";
+import { FilterScopes, VacationData } from "types";
+import useEditorContentStyles from "styles/editor-content/editor-content";
+import Holidays from "date-holidays";
+import DateRangePicker from "components/generics/date-range-picker/date-range-picker";
+
+interface VacationRequestFormProps {
+  onClick: () => void;
+  buttonLabel: string;
+  vacationData: VacationData;
+  setVacationData: (vacationData: VacationData) => void;
+}
+
+/**
+* Form component for vacation requests
+* @param param0 
+*/
+const VacationRequestForm = ({
+  onClick,
+  buttonLabel,
+  vacationData,
+  setVacationData
+}: VacationRequestFormProps) => {
+  const classes = useEditorContentStyles();
+  const dateFormat = "yyyy.MM.dd";
+  const [ datePickerView ] = useState<CalendarPickerView>("day");
```

Comment:
As this state value is never being updated or changed (as there is no setDatePickerView) then it doesn't need to be state. You can just pass in the "day" string wherever datePickerView is being used.